### ISSUE TITLE
Create a Interface Class for our backend

### ISSUE THREAD

**Issue by AaryanKapoor22** [MineGamer69/ParkingApp#3]
Since we will be using databases / text files to store data I think we can make it more efficient by creating a interface class for future expansion. I think that would be a really easy way to link and use multiple databases, off of one interface.

**Comment by atoomer** [comment]
Good idea, especially if we intend to store even more data in the future.

**Comment by NCRaider1** [comment]
I think this is a very interesting idea, as stated, it allows for possibly more data in the future with less things to overwrite. It could make an expansion possible, especially if additional features are added, like selected times slots, reserved parking, etc.
